src/util.c

```diff
@@ -49,6 +49,47 @@ char *path_append(const char *path, const char *filename)
     return str;
 }
 
+char *escape_spaces(const char *path)
+{
+    char space = ' ';
```

Comment:
We don't need this variable. Remove please.

Reply:
Commit updated.